utils/update_checkout/update-checkout-config.json

```diff
@@ -167,7 +170,8 @@
                 "wasmkit": "0.1.2",
                 "curl": "curl-8_9_1",
                 "libxml2": "v2.11.5",
-                "zlib": "v1.3.1"
+                "zlib": "v1.3.1",
+                "mimalloc": "v3.0.1"
```

Comment:
Can you add this to the next and rebranch scheme too?

Reply:
Whoops, absolutely!